pkg/pool-weighted/contracts/smart/WeightCompression.sol

```diff
@@ -23,6 +23,8 @@ import "@balancer-labs/v2-solidity-utils/contracts/math/FixedPoint.sol";
  */
 library WeightCompression {
```

Comment:
Let's make sure we tackle #685 reasonably soon

Reply:
I was going to use WordCodec as an example, but we don't have that one either